docs/sourcemap-api.asciidoc

```diff
@@ -18,26 +16,19 @@ http(s)://{hostname}:{port}/v1/rum/sourcemaps
 [[sourcemap-request-fields]]
 [float]
 ==== Request Fields
-The request consists of some meta information and the actual source map.
-The meta information must contain the following fields:
+The request must include some fields needed to identify `source map` correctly later on:
 
 * `service_name`
 * `service_version`
 * `bundle_filepath`: needs to be the absolute path of the final bundle as it is used in the web application
 
-The meta information is used to identify a `source map` when source mapping is applied.
-
-The actual source map must be attached to the request as a `file upload`
-and it must match the specification for 
-https://docs.google.com/document/d/1U1RGAehQwRypUTovF1KRlpiOFze0b-_2gc6fAH0KY0k[Source map revision 3 proposal].
-
+The source map must follow the
+https://docs.google.com/document/d/1U1RGAehQwRypUTovF1KRlpiOFze0b-_2gc6fAH0KY0k[Source map revision 3 proposal] spec and be attached as a `file upload`.
 
 [[sourcemap-api-examples]]
 [float]
 ==== Example
 
-Send an example source map to the APM Server:
```

Comment:
Is this text not accurate? IMO, it's nice to introduce examples for people who scan stuff quickly.